lib/src/widgets/countdown_clock.dart

```diff
@@ -55,6 +59,7 @@ const _emergencyDelay = Duration(seconds: 20);
 class _CountdownClockState extends ConsumerState<CountdownClock> {
   Timer? _timer;
   Duration timeLeft = Duration.zero;
+  bool _emergencySoundEnabled = true;
```

Comment:
You don't need that local property since it doesn't change. Use directly `widget.emergencySoundEnabled` instead.